src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
```

Comment:
Yes as it was before i think it was ok.

I liked a lot the "trasparent" replacement of rgba(0,0,0,0) you did, are we sure is it supported as well and even better than rgba()? 

I tried searching a lot but no results, probably trasparent is fully supported since CSS1 so better than rgba() in such cases....